docs/user/03-10-preconfigured-secret.md

```diff
@@ -0,0 +1,50 @@
+# Preconfigured Credentials and Access
+
+When you create SAP BTP, Kyma runtime, all necessary resources for consuming SAP BTP services are created, and the basic cluster access is configured.
+
+## Credentials
+
+When you create a Kyma instance in the SAP BTP cockpit, the following events happen in your subaccount:
+1. An SAP Service Manager service instance with the `service-operator-access` plan is created.
+2. An SAP Service Manager service binding with access credentials for the SAP BTP Operator is created.
+3. The credentials from the service binding are passed on to the Kyma service instance in the creation process.
+4. The `sap-btp-manager` Secret is created and managed in the `kyma-system` namespace.
+5. The SAP BTP Operator module is installed by default together with:
+   * The `sap-btp-manager` Secret.
+   * The `sap-btp-service-operator` Secret with the access credentials for the SAP BTP service operator. You can view the credentials in the `kyma-system` namespace.
+   * The `sap-btp-operator-config` ConfigMap.
+
+> [!TIP]
+> In this scenario, the `sap-btp-service-operator` Secret is automatically generated when you create Kyma runtime. To create this Secret manually for a specific namespace, see [Create an SAP BTP Service Operator Secret](./tutorials/04-20-create-btp-service-operator-secret.md).
+
+The `sap-btp-manager` Secret provides the following credentials:
+* **clientid**
+* **clientsecret**
+* **cluster_id**
+* **sm_url**
+* **tokenurl**
+
+When you add the SAP BTP Operator module to your cluster, the `sap-btp-manager` Secret generates the SAP BTP service operator's resources as shown in the following diagram:
+<!-- for the HP doc this sentence is different: The SAP BTP Operator module is added by default to your cluster and the `sap-btp-manager` (...) -->
+
+![module_credentials](../assets/module_credentials.drawio.svg)
```

Comment:
In the diagram, I'm not sure if I understand right which element passes the credentials.